test/functional/inspec_vendor_test.rb

```diff
@@ -25,6 +25,8 @@
       out = inspec('vendor --overwrite', "cd #{dir} &&")
       out.stderr.must_equal ''
       out.exit_status.must_equal 0
+      # this fixes the osx /var symlink to /private/var causing this test to fail
+      out.stdout.gsub!(%r{\/private\/var}, '/var')
```

Comment:
I don't even think you need to do this as a regex. `gsub` and `gsub!` accept a string match too.

```
[1] pry(main)> 'Something blah /private/var/dir_a/dir_b/private/blah more stuff'.gsub('/private/var', '/var')
=> "Something blah /var/dir_a/dir_b/private/blah more stuff"
```